fhir-client/src/main/java/com/ibm/fhir/client/impl/FHIRClientImpl.java

```diff
@@ -768,6 +768,7 @@ protected synchronized Client getClient() throws Exception {
             // Add a hostname verifier if we're using an ssl transport.
             if (usingSSLTransport() && !isHostnameVerificationEnabled()) {
                 cb = cb.hostnameVerifier(new HostnameVerifier() {
+
```

Comment:
Is there another change here?

Reply:
ah, sorry, this file is added by mistake...

Reply:
no problem - just confirming.